### ISSUE TITLE
Weird crash on launch through RDP session

### ISSUE THREAD

**Issue by FaceCrap** [Willy-JL/F95Checker#43]
I haven't had this happen before, but then again, it's always been already running when I remote into my desktop on mobile.
See log below
```
Traceback (most recent call last):
File C:\hostedtoolcache\windows\Python\3.11.0\x64\Lib\site-pack line 138, in run
File C:\hostedtoolcache\windows\Python\3.11.0\x64\Lib\site-pack line 16, in run
File D:\a\F95Checker\F95Checker\main.py line 72, in <module>
File D:\a\F95Checker\F95Checker\main.py line 46, in main
File D:\a\F95Checker\F95Checker\modules\gui.pyline 591, in main_loop
File C:\hostedtoolcache\windows\Python\3.11.0\x64\Lib\site-pack line 113, in process_inputs
File C:\hostedtoolcache\windows\Python\3.11.0\x64\Lib\site-pack line 11, in compute_fb_scale
ZeroDivisionError: float division by zero
```
Bizar, Just got home, launched it on the desktop, and it started like nothing the matter... 

This isn't the first time I experience weirdness accessing F95Checker through Microsoft's iOS RD Client, When it is already running, I can't get it to respond to the mouse either...

**Comment by FaceCrap** [comment]
Well, paint me purple and call me silly... Just read the post on F95Zone re build 691. Tried it again through remoting in... and it started ... kept trying a couple of times, no issue...

Then I got the notification for build 698... tried that a couple of times... first few times nothing, both exe versions just launched, 

Then suddenly the regular exe bombed again during launch and threw the same error with the exact same line numbers in a dialogue popup (just like 691) and nothing in the log file... but the debug exe just ran...

After that the regular exe just kept throwing that same error until I ran 691 again, which launched, and this may be coincidence, but I ran 698 again and suddenly the regular exe launched fine. Frankly I'm baffled... I've got no idea why it happens or what triggers it.

**Comment by Willy-JL** [comment]
Looks like a simple distraction error by the pyimgui team, I will submit a pull request shortly.

The issue:
```py
def compute_fb_scale(window_size, frame_buffer_size):
    win_width, win_height = window_size
    fb_width, fb_height = frame_buffer_size

    # future: remove floats after dropping py27 support
    if win_width != 0 and win_width != 0:  # ISSUE HERE
        return float(fb_width) / win_width, float(fb_height) / win_height

    return 1., 1.
 ```

here they divide the framebuffer dimensions by the window dimensions, and logically if either of the window dimensions is 0 the division is skipped because a division by 0 will cause an error. however the check for 0ed dimensions checks both times on the width, never on the height

**Comment by Willy-JL** [comment]
Released in [version 9.5](https://github.com/Willy-JL/F95Checker/releases/tag/9.5)
